acquia_lift.admin.inc

```diff
@@ -287,13 +287,36 @@ function acquia_lift_admin_form_submit($form, &$form_state) {
  * @param stdClass $option_set
  *   (optional) The content variation to show in the default view.
  */
-function acquia_lift_report($form, &$form_state, $agent_data, $option_set = NULL) {
+function acquia_lift_report($form, &$form_state, $agent_data, $option_set = NULL, $nested = FALSE) {
   if (!$agent = personalize_agent_load_agent($agent_data->machine_name)) {
```

Comment:
Yeah, I found it weird too, but then discovered that it's needed further down - it's the instantiated object and we need to call its errors() method further down. Going to rename the variable to `$agent_instance` to make it clearer and move it further down to where it's actually needed, with an explanation.